Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "IntervalConstraintProgramming"
+uuid = "138f1668-1576-5ad7-91b9-7425abbf3153"
+version = "0.11"
+
+[compat]
+IntervalArithmetic = "≥ 0.15.0"
+IntervalContractors = "≥ 0.3.0"
+IntervalRootFinding = "≥ 0.4.0"
+MacroTools = "≥ 0.4.0"
+julia = "≥ 1.0.0"
```

Comment:
Aren't we requiring Julia 1.1? And ModelingToolkit 0.2

Reply:
Oh Right, Sorry its by mistake.
But Isn't for ModelingToolkit it should be autogenerated ?

Reply:
I guess it's because you hadn't put a lower bound in to REQUIRE.
Just add it by hand.